File: microservices/investment/src/main/java/com/illunex/invest/investment/persistence/entity/EvaluateReviewItem.java
package com.illunex.invest.investment.persistence.entity;

import lombok.*;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Table(name = "evaluate_review_item")
public class EvaluateReviewItem {
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long idx;
    String category;
    String content;
    Integer point;
    LocalDateTime updateDate;

    @ManyToOne @JoinColumn(name = "judge_idx")
    EvaluateJudge judge;
       
}

File: microservices/invest/src/main/kotlin/com/illunex/invest/invest/persistence/entity/EvaluateReviewItem.kt
package com.illunex.invest.invest.persistence.entity

import javax.persistence.*

@Entity
@Table(name = "evaluate_review_item")
data class EvaluateReviewItem(
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    var idx: Long? = null,
    var title: String? = null,
    var point: Int? = 0,
    var content: String? = null,
    var updateDate: String? = null,
    var deleted: Boolean? = false,

    @ManyToOne @JoinColumn(name = "category_idx")
    var category: EvaluateReviewItemCategory?

) {
    constructor() : this(null, null, null, null, null, null, null)
}
